rest-api-spec/src/main/resources/rest-api-spec/test/search/30_limits.yml

```diff
@@ -85,18 +85,22 @@ setup:
 ---
 "Docvalues_fields size limit":
 
+  - skip:
+      version: " - 7.99.99"
+      reason: waiting for backport
```

Comment:
I dislike having to do this here just because I changed the error message slightly. Happy do get some better ideas what we do in such cases in other places if possible.

Reply:
For me the old message was just as clear, and lets us avoid having to update the test. But if we want to keep it, I wonder if we could just remove this test since it's already covered by `SearchServiceTests#testMaxDocvalueFieldsSearch`?